extractor/src/main/java/org/schabi/newpipe/extractor/services/soundcloud/SoundcloudChannelExtractor.java

```diff
@@ -56,16 +60,25 @@ public String getName() {
     }
 
     @Override
-    public String getAvatarUrl() {
-        return user.getString("avatar_url");
+    public List<Image> getAvatars() {
+        List<Image> images = new ArrayList<>();
+
+        images.add(new Image(user.getString("avatar_url"), Image.LOW, Image.LOW));
```

Comment:
there is also a small one: small.jpg

We therefore need a third image size constant